pkgs/games/duckmarines/default.nix

```diff
@@ -0,0 +1,60 @@
+{ stdenv, fetchurl, unzip, love, lua, makeWrapper, makeDesktopItem }:
+
+let
+  pname = "duckmarines";
+  version = "1.0b";
+
+  icon = fetchurl {
+    url = "http://tangramgames.dk/img/thumb/duckmarines.png";
+    sha256 = "07ypbwqcgqc5f117yxy9icix76wlybp1cmykc8f3ivdps66hl0k5";
+  };
+
+  desktopItem = makeDesktopItem {
+    name = "duckmarines";
+    exec = "${pname}";
+    icon = "${icon}";
+    comment = "Duck-themed action puzzle video game"; 
+    desktopName = "Duck Marines";
+    genericName = "duckmarines";
+    categories = "Game;";
+  };
+
+in 
+
+stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/SimonLarsen/${pname}/releases/download/v${version}/${pname}-1.0-love.zip";
+    sha256 = "0fpzbsgrhbwm1lff9gyzh6c9jigw328ngddvrj5w7qmjcm2lv6lv";
+  };
+
+  nativeBuildInputs = [ lua love ];
```

Comment:
Both `lua` and `love` are actually needed to run the game and not to build it therefore both are normal `buildInputs` and belong there. `makeWrapper` on the other side can be placed in `nativeBuildInputs`.